route.proto

```diff
@@ -49,7 +76,7 @@ message RouteCreate {
 // - `status` to `ROUTE_STATUS_DROPPING_OFF`
 message RouteDepotIn {
   required uint64 id = 1;
-  required double total_weight = 2;
+  required double waste_weight = 2;
```

Comment:
we have waste weight on status, do we need that on route model too?

Reply:
Well it's optional)